src/SFML/Network/IpAddress.cpp

```diff
@@ -81,6 +86,7 @@ std::optional<IpAddress> IpAddress::resolve(std::string_view address)
         return IpAddress(ntohl(ip));
     }
 
+    // Not generating en error message here to keep this function lightweight.
```

Comment:
Not sure if "lightweight" is a good argument for not logging errors.

Why is the context information in the situation not useful?

Reply:
I agree that this comment is kinda bad. The reason why I don't want to include an error message here is because `resolve` returning an empty optional is not really a failure, it might be a totally fine scenario if we are trying to resolve user-entered addresses.

What I am trying to say is that this is one of the situations where "empty optional" does not mean "unexpected failure" but rather "possibly expected failure to parse".